packages/cli/src/commands/items/list.ts

```diff
@@ -25,8 +63,18 @@ export default class ItemsList extends MeecoCommand {
       this.error('Must specify a valid auth config file');
     }
 
+    const filters: IItemListFilterOptions = {
+      templateIds,
```

Comment:
If you go with the suggestion below then this would change to `templateIds: templateIds.split(',')`

Reply:
Actually it seems like `flags.string` supports a `multiple` option too so that's another alternative - change it to `templateId` and `multiple` so you would do something like (I assume)

`items:get --templateId "template_id_1" --templateId "template_id_2"`

Reply:
https://oclif.io/docs/flags

Reply:
Alternatively you could just make all of them accept comma separated args by default, and don't have both `classificationNodeName` and `classificationNodeNames`

Reply:
- let me find out oclif flags option.